devtools/create_xeen/constants.cpp

```diff
@@ -1711,6 +1713,8 @@ static const char* const BUY_X_FOR_Y_GOLD =
 static const char* const SELL_X_FOR_Y_GOLD =
   "\x3""l\v000\t000\fd�த��� %s\fd �� %lu �����%s?";
 
+static const char *const RU_SELL_X_FOR_Y_GOLD_ENDINGS[2] = { "��", "��" };
```

Comment:
Same problem as before. Any non-ASCII chars should be encoded using \x?? sequences rather than being directly included.

Reply:
Fixed in later commits